l10n_it_fatturapa_out/readme/USAGE.rst

```diff
@@ -4,10 +4,14 @@
  * Selezionare 1 o N fatture ed eseguire la procedura guidata "Esporta fattura elettronica"
  * Per le fatture estere, è possibile inviarle a soli fini fiscali inserendo il codice identificativo XXXXXXX (7 volte X) ed avendo cura di indicare il paese del partner.
    Le fatture vanno comunque spedite al cliente, ma si evita la predisposizione dell'esterometro.
+ * É possibile esportare le fatture cliente con le righe articolo con un CodiceTipo diverso dallo standard 'ODOO' creando un parametro 'fatturapa.codicetipo.odoo' (in Configurazione > Funzioni tecniche > Parametri > Parametri di sistema) con il codice voluto (tipicamente su richiesta del cliente).
+   Non è possibile impostare un diverso CodiceTipo per cliente, al momento.
```

Comment:
@sergiocorato grazie. Secondo me questa parte va in `CONFIGURE`

Reply:
@eLBati fatto